save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/controller/HeartbeatController.kt

```diff
@@ -52,16 +52,10 @@ class HeartbeatController(private val agentService: AgentService,
         )
             .then(
                 when (heartbeat.state) {
-                    AgentState.IDLE -> agentService.getNewTestsIds(heartbeat.agentId).subscribeOn(scheduler)
+                    AgentState.IDLE -> agentService.getNewTestsIds(heartbeat.agentId)
```

Comment:
Why did you remove `subscribeOn`?

Reply:
Because it's a little refactored now: before we were adding a blocking handler for the returned Mono, which was executed on `sheduler`. Now we make another request inside `doOnSuccess` and subscribe on *that* request on `scheduler`.